src/release.sh

```diff
@@ -1,23 +1,47 @@
 #!/usr/bin/env bash
 set -e
 
-# This script will do a release of the artifact according to http://maven.apache.org/maven-release/maven-release-plugin/
-git config --global user.email "openbankbot@forgerock.com";
-git config --global user.name "openbankbot";
+# avoid the release loop by checking if the latest commit is a release commit
+readonly local last_release_commit_hash=$(git log --author="$GIT_RELEASE_BOT_NAME" --pretty=format:"%H" -1)
```

Comment:
I think I need to get the GIT repo first. Just don't know how I can do that.
I guess by mounting a folder in the docker instance?